package.json

```diff
@@ -7,9 +7,9 @@
     "coverage-html-report": "rm -rf coverage/ && nyc report --reporter=html && opn coverage/index.html",
     "generate-constants": "./scripts/generate-constants.js",
     "install": "node scripts/install.js",
```

Comment:
Normally you would do the `node-gyp-build` command in the `install` script, but we have another seam there due to the system core.